src/styles/blocks/card.scss

```
@@ -0,0 +1,93 @@
+.card {
+  width: #{$card-width};
```

```suggestion
  width: $card-width;
```
no need to add '# {}' unless it is in an expression state